packages/lwc-language-server/src/javascript/type-mapping.ts

```diff
@@ -0,0 +1,560 @@
+import {
```

Comment:
This module transforms new metadata (from `@lwc/metadata`) into the data-structure produced by very old versions of the LWC compiler.